Resources/config/services.xml

```diff
@@ -40,6 +40,10 @@
             <tag name="monolog.processor" />
         </service>
 
+        <service id="Monolog\Processor\IntrospectionProcessor" class="Monolog\Processor\IntrospectionProcessor" public="false">
```

Comment:
@aallier @loverg-c we will need to check that this does not have perf impact to call that function to get debug trace: https://github.com/Seldaek/monolog/blob/main/src/Monolog/Processor/IntrospectionProcessor.php#L67C18-L67C33